zookeeper-examples/src/main/java/MainClassNode1.java

```diff
@@ -0,0 +1,61 @@
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Vertx;
+import io.vertx.core.VertxOptions;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.spi.cluster.ClusterManager;
+import io.vertx.spi.cluster.zookeeper.ZookeeperClusterManager;
+
+import java.net.UnknownHostException;
+
+/*
+    Represents Vert.x instance running on a node (node1)
+    This class deploys sender verticle on node1 (machine1)
+ */
+public class MainClassNode1 {
+    public static void main(String[] args) throws UnknownHostException {
+        JsonObject zkConfig = configureClusterManager();
+        ClusterManager zookeeperClusterManager = new ZookeeperClusterManager(zkConfig);
+
+        VertxOptions options = configureVertx(zookeeperClusterManager);
+        Vertx.clusteredVertx(options, res -> {
+            if (res.succeeded()) {
+                Vertx vertx = res.result();
+                DeploymentOptions deploymentOptions = new DeploymentOptions().setInstances(2);
+                vertx.deployVerticle("verticle.SenderVerticle", deploymentOptions);
+            }
+        });
+    }
+
+    /**
+     * create configuration object to be used to configure the Zookeeper cluster manager
+     * @return JsonObject representing Zookeeper configuration
+     */
+    private static JsonObject configureClusterManager() {
+        // add the IP of the machine hosting the cluster manager
+        JsonObject zkConfig = new JsonObject();
+        zkConfig.put("zookeeperHosts", "192.168.1.12");
```

Comment:
Can't it be `localhost`

Reply:
Ok, but for the user running the example on a laptop, how would that work? Editing the code to enter the IP is not very user-friendly. We can add a comment explaining how to configure it for more "production" use cases.

Reply:
Do you prefer comments in code, or add a section in readme?

Reply:
Good question. I don't have a strong opinion. I like having things in the code so the user can quickly understand where to _patch_ during a copy-paste (as we all do)

Reply:
Ok, I will add these modifications and explain how to run Zookeeper and update the PR.